matchbox_socket/src/webrtc_socket/wasm.rs

```diff
@@ -325,12 +331,12 @@ async fn complete_handshake(
         try_add_rtc_ice_candidate(&conn, &candidate).await;
     }
 
-    // select for channel ready or ice candidates
+    // select for data channels ready or ice candidates
     debug!("waiting for data channels to open");
     loop {
         select! {
-            _ = wait_for_channels => {
-                debug!("channel ready");
+            _ = data_channels_ready_fut => {
+                debug!("data channels ready");
```

Comment:
this strikes me as a trace event

Reply:
Personally I'm pretty interested in this message, more than I would be in typical trace messages. It's the point where we know the peer connection was successful.